apps/src/p5lab/spritelab/commands/actionCommands.js

```diff
@@ -45,6 +62,20 @@ export const commands = {
     });
     return touching;
   },
+
+  isTouchingSprite(spriteArg, targetArg) {
+    let sprites = coreLibrary.getSpriteArray(spriteArg);
+    let targets = coreLibrary.getSpriteArray(targetArg);
+    let touching = false;
+    sprites.forEach(sprite => {
+      targets.forEach(target => {
+        if (sprite.isTouching(target)) {
+          touching = true;
```

Comment:
nit: if you use [`for...in`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in) instead of `forEach` here, you can break early